pkg/app/daemon/daemon.go

```diff
@@ -34,6 +37,15 @@ const (
 	// to prepend the content of `RebaseTODOEnvKey` to the default `git-rebase-todo`
 	// file instead of using it as a replacement.
 	PrependLinesEnvKey string = "LAZYGIT_PREPEND_LINES"
+
+	// If this is set, it tells lazygit to read the original todo file, and
+	// change the action for one or more entries in it. The value of the variable
+	// will have one or more lines of the form "Sha1:newAction", e.g.
+	// a02b54e1b7e7e8dd8bc1958c11ef4ee4df459ea4:edit
+	// The existing action of the todo to be changed is expected to be "pick".
+	//
+	// If this is used, the value of RebaseTODOEnvKey must be empty.
+	ChangeTodoActionEnvKey string = "LAZYGIT_CHANGE_TODO_ACTION"
```

Comment:
We've got a few env vars now, and it's not ideal working with an interface that consists of multiple env vars. What are your thoughts on having a struct that contains all the args we need and JSONifying it into a single env var then converting back to a struct here. That allows for arbitrary complexity in the interface

Reply:
Yes, good idea. I squashed this into the last commit (see ea2583ef590), using fixups didn't seem to make much sense for this.